functions/src/Services/Event.ts

```diff
@@ -2,19 +2,26 @@ import { User } from "../Models/User";
 import { Event, eventConverter } from "../Models/Event";
 import { firestore } from 'firebase-admin';
 
+/**
+ * This function is used to easily handle retrieving the event collection with the converter
+ */
+function getEventCollection() {
+	return firestore().collection('events').withConverter(eventConverter);
+}
+
 export const createEvent = (event: Event) => {
-    const eventCollection = firestore().collection('events').withConverter(eventConverter);
+	const eventCollection = getEventCollection();
 	eventCollection.add(event);
 };
 
```

Comment:
Can be one line if `getEventCollection` is a variable

Reply:
did this for consistency with other functions

Reply:
Yeah consistency, also sorry if I spam your email, I'm writing comments on all the changes to make sure I did not forget any